plugins/feedback/src/components/FeedbackTable/FeedbackTable.tsx

```diff
@@ -0,0 +1,296 @@
+import React, { useState } from 'react';
+import { useDebounce } from 'react-use';
+
+import { parseEntityRef } from '@backstage/catalog-model';
+import {
+  SubvalueCell,
+  Table,
+  TableColumn,
+  useQueryParamState,
+} from '@backstage/core-components';
+import { useApi } from '@backstage/core-plugin-api';
+import {
+  EntityPeekAheadPopover,
+  EntityRefLink,
+} from '@backstage/plugin-catalog-react';
+
+import {
+  Chip,
+  IconButton,
+  Link,
+  makeStyles,
+  Paper,
+  TableContainer,
+  TextField,
+  Theme,
+  Tooltip,
+  Typography,
+} from '@material-ui/core';
+import BugReportOutlined from '@material-ui/icons/BugReportOutlined';
+import Clear from '@material-ui/icons/Clear';
+import Search from '@material-ui/icons/Search';
+import TextsmsOutlined from '@material-ui/icons/TextsmsOutlined';
+
+import { feedbackApiRef } from '../../api';
+import { FeedbackType } from '../../models/feedback.model';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  textField: {
+    padding: 0,
+    margin: theme.spacing(2),
+    width: '70%',
+    [theme.breakpoints.up('lg')]: {
+      width: '30%',
+    },
+  },
+}));
+
+export const FeedbackTable: React.FC<{ projectId?: string }> = (props: {
+  projectId?: string;
+}) => {
+  const projectId = props.projectId ? props.projectId : 'all';
+  const api = useApi(feedbackApiRef);
+  const [feedbackData, setFeedbackData] = useState<FeedbackType[]>([]);
+  const [tableConfig, setTableConfig] = useState({
+    totalFeedbacks: 100,
+    page: 1,
+    pageSize: 5,
+  });
+  const [queryState, setQueryState] = useQueryParamState<string>('id');
+  const [loading, setLoading] = useState(true);
+  const [searchText, setSearchText] = useState<string>('');
+  const classes = useStyles();
+
+  const columns: TableColumn[] = [
+    {
+      width: '1%',
+      field: 'feedbackType',
+      title: 'Type',
+      render: (row: any) => {
+        const data: FeedbackType = row;
+        return data.feedbackType === 'BUG' ? (
+          <BugReportOutlined color="secondary" key={data.feedbackType} />
+        ) : (
+          <TextsmsOutlined color="primary" key={data.feedbackType} />
+        );
+      },
+    },
+    {
+      field: 'summary',
+      title: 'Summary',
+      render: (row: any) => {
+        const data: FeedbackType = row;
+        const getSummary = () => {
+          if (data.summary.length > 100) {
+            if (data.summary.split(' ').length > 1)
+              return `${data.summary.substring(
+                0,
+                data.summary.lastIndexOf(' ', 100),
+              )}...`;
+            return `${data.summary.slice(0, 100)}...`;
```

Comment:
Is the full value of this field shown somewhere should this case arise?

Reply:
Full summary is shown when user clicks on feedback in table

![feedbackDetails](https://private-user-images.githubusercontent.com/42431299/294843468-5618a0fb-060c-4763-866b-e29e028cb246.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ3MDM3NTgsIm5iZiI6MTcwNDcwMzQ1OCwicGF0aCI6Ii80MjQzMTI5OS8yOTQ4NDM0NjgtNTYxOGEwZmItMDYwYy00NzYzLTg2NmItZTI5ZTAyOGNiMjQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA4VDA4NDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY0MDUyMDBmOGE5Yjg1MDczZWI5MjBjMTJhZmYzZDI3MWVjODM0MDg2M2JkYzMyZGNmYjE1OTI3M2E0MTM0YWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.m29ooIPPoTl9k72hn-nWDHC64ofq4BGbym_53CZxczE)